packages/forma-36-react-components/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import cn from 'classnames';
+import Heading from '../Typography/Heading';
+import Paragraph from '../Typography/Paragraph';
+
+const styles = require('./EmptyState.css');
+
+export type EmptyStateProps = {
+  /**
+   * Class names to be appended to the className prop of the component
+   */
+  className?: string;
+  /**
+   * Child nodes to be rendered in the component
+   */
+  children?: React.ReactNode;
+  /**
+   * An ID used for testing purposes applied as a data attribute (data-test-id)
+   */
+  testId?: string;
+  /**
+   * Props for image block
+   */
+  image?: ImageProps;
+  /**
+   * Heading text
+   */
+  heading: string;
+  /**
+   * Description text
+   */
+  description: string;
+} & typeof defaultProps;
+
+interface ImageProps {
+  /**
+   * Url of the illustration for empty state
+   */
+  url: string;
+  /**
+   * Width of the illustration for empty state
+   */
+  width: string;
+  /**
+   * Height of the illustration for empty state
+   */
+  height: string;
+  /**
+   * Image description
+   */
+  description: string;
+  /**
+   * Image class
+   */
+  className?: string;
+}
+
+const defaultProps = {
+  testId: 'cf-ui-empty-state',
+};
+
+export class EmptyState extends Component<EmptyStateProps> {
+  static defaultProps = defaultProps;
+
+  render() {
+    const {
+      className,
+      children,
+      testId,
+      image,
+      heading,
+      description,
+      ...otherProps
+    } = this.props;
+    const classNames = cn(styles.EmptyState, className);
+
+    return (
+      <div {...otherProps} className={classNames} data-test-id={testId}>
+        <div className={styles['EmptyState_container']}>
+          {image && (
+            <div
+              style={{
+                backgroundImage: `url(${image.url})`,
+                width: image.width,
+                height: image.height,
+              }}
+              className={cn(
+                image.className,
+                styles['EmptyState_element'],
+                styles['EmptyState_illustration'],
+              )}
+              aria-label={image.description}
+              role="img"
+            />
+          )}
+          <Heading element={'h1'} className={styles['EmptyState_element']}>
```

Comment:
You can remove the `element` prop here as h1 is the default. Then I'd recommend adding a `headingProps` object to allow developers to override the default element if needed.

Reply:
I don't think it's necessary, the design I got from Sara has consistent heading and paragraph sizings, the only customization I could trace is different image and child components - buttons or other controls, for example. I'll remove the element prop though.

Reply:
@thebesson Ahh, this prop doesn't have any impact on visual appearance - it just changes the semantic element used to render the heading. It's useful if the title of the page is rendered in a H1 and using this component as a H1 is incorrect semantically.

Reply:
I see. Makes sense, but I don't want to overcomplicate the api for this component. Semantically speaking, if it's a state for when this component is the only thing displayed, does it makes sense to have it as H1?

Reply:
Oop - tried replying via email but it didn't make this thread. Reposting here for better clarity.

If the containing element is an `article` then H1 would be fine. Maybe this would be a
good change anyway. The prop would be optional anyway though so wouldn’t over
complicate the api in my opinion. I quite strongly believe this change should be done, as we should have the convention that it's possible to pass props to all child f36 components